folium/folium.py

```diff
@@ -676,11 +663,9 @@ def _popup_render(self, popup=None, mk_name=None, count=None,
             Type of marker. Simple, Circle, etc.
         count: int, default None
             Count of marker
-        popup_on: boolean, default True
-            If False, no popup will be rendered
         """
-        if not popup_on:
-            return 'var no_pop = null;'
+        if not popup:
+            return ''
```

Comment:
> In fact if there is no popup text, I do not set a javascript variable at all in the HTML template.

In a local test the "click" hand appears when hovering a mark making the user think that he can click and see something.  I guess we do need to set `'var no_pop = null;'`.

![popuptest](https://cloud.githubusercontent.com/assets/950575/7840026/e6965df6-046b-11e5-82a0-67f20d1a073d.png)